lib/version.js

```diff
@@ -282,25 +282,34 @@ function _commit (version, localData, cb) {
   const options = { env: process.env }
   const message = npm.config.get('message').replace(/%s/g, version)
   const sign = npm.config.get('sign-git-tag')
-  const flag = sign ? '-sm' : '-am'
-  BB.join(
-    addLocalFile('package.json', options, false),
-    localData.hasShrinkwrap &&
-    addLocalFile('npm-shrinkwrap.json', options, true),
-    localData.hasPackageLock &&
-    addLocalFile('package-lock.json', options, true)
-  ).then(() => {
+  const flagForTag = sign ? '-sm' : '-am'
+
+  stagePackageFiles(localData, options).then(() => {
     return git.exec([ 'commit', '-m', message ], options)
   }).then(() => {
     if (!localData.existingTag) {
       return git.exec([
         'tag', npm.config.get('tag-version-prefix') + version,
-        flag, message
+        flagForTag, message
       ], options)
     }
   }).nodeify(cb)
 }
 
+function stagePackageFiles (localData, options) {
+  return new BB(function (resolve, reject) {
```

Comment:
I'm doing this right now! Is there a possibility that this could be merged to the 5.3.0 relese today?